### PR TITLE
Updated links to new Best Practices section of dataverse.org [ref #4544]

### PR DESCRIPTION
**Pull request by mheppler**
## Related Issues

- connects to #4544 Replace links to best-practices.dataverse.org with links to dataverse.org

## Pull Request Checklist

- [ ] Unit [tests][] completed
- [ ] Integration [tests][]: None
- [ ] Deployment requirements, [SQL updates][], [Solr updates][], etc.: None
- [x] [Documentation][docs] completed
- [x] Merged latest from "develop" [branch][] and resolved conflicts

[tests]: http://guides.dataverse.org/en/latest/developers/testing.html
[SQL updates]: https://github.com/IQSS/dataverse/tree/develop/scripts/database/upgrades
[Solr updates]: https://github.com/IQSS/dataverse/blob/develop/conf/solr/4.6.0/schema.xml
[docs]: http://guides.dataverse.org/en/latest/developers/documentation.html
[branch]: http://guides.dataverse.org/en/latest/developers/branching-strategy.html

### CODE REVIEW

File: src/main/java/Bundle.properties [hunk 1]
```diff
@@ -1375,9 +1375,9 @@ file.dataFilesTab.terms.list.termsOfUse.header=Terms of Use
...
 file.dataFilesTab.terms.list.termsOfUse.no.waiver.txt=No waiver has been selected for this dataset.
-file.dataFilesTab.terms.list.termsOfUse.waiver.txt.description=Our <a href="http://best-practices.dataverse.org/harvard-policies/community-norms.html" title="Dataverse Community Norms - Dataverse Best Practices" target="_blank">Community Norms</a> as well as good scientific practices expect that proper credit is given via citation. Please use the data citation above, generated by the Dataverse.
+file.dataFilesTab.terms.list.termsOfUse.waiver.txt.description=Our <a href="https://dataverse.org/best-practices/dataverse-community-norms" title="Dataverse Community Norms - Dataverse.org" target="_blank">Community Norms</a> as well as good scientific practices expect that proper credit is given via citation. Please use the data citation above, generated by the Dataverse.
```
Comment (pdurbin) [inline src/main/java/Bundle.properties]: I'm still seeing some "best-practices" (lower down) on `file.dataFilesTab.terms.list.termsOfUse.termsOfUse.description`

Comment (mheppler) [reply src/main/java/Bundle.properties]: Good catch.

Comment (pdurbin) [reply src/main/java/Bundle.properties]: Thanks. I recommend ack: https://beyondgrep.com

**Comment by mheppler** [comment]
@pdurbin -- last link updated. Didn't double check my find and replace with a second search in the project before committing. Have now confirmed there are no more references to "best-practices.dataverse.org". Ready again for review and QA.

**Pull request merged**